### DISCUSSION TITLE
Name for the ARrow's Mascot

### DISCUSSION THREAD

**Discussion by UBA-GCOEN** [UBA-GCOEN/ARrow#38] [Polls]
Let's decide the name for our Mascot 🤖

![image](https://github.com/UBA-GCOEN/ARrow/assets/77800620/8db3724a-4eff-4c2d-bca9-79676bfd5691)

**Comment by UBA-GCOEN** [comment]
@Siddhant-Patil0203  @vishal10kesharwani @Naresh-chandanbatve @LadeHarshal 
Please Acknowledge this poll🙂

**Comment by Siddhant-Patil0203** [comment]
I personally like the name **Guido** as it resonates with the word guide 
also, a fun fact 😁 : **_Guido van Rossum_**  is the father of the Python programming Language

**Comment by LadeHarshal** [comment]
I liked the name Whiz. 
In dragon ball anime , Whis is the attendant for the god of destruction 
Also funfact whis has the same color as our bot
![image](https://github.com/UBA-GCOEN/ARrow/assets/92210851/aa6291fe-4da7-4f72-a0c8-09fc5843f536)
